artifacts/artifact.py

```diff
@@ -86,3 +86,40 @@ def AppendSource(self, type_indicator, attributes):
 
     self.sources.append(source_object)
     return source_object
+
+  def AsDict(self):
+    """Converts artifact to dict from ArtifactDefinition
```

Comment:
It does not convert since the original is preserved.

"""Represents an artifacts as a dictionary.

Returns:
   A dictionary containing the artifact attributes.
"""